src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Post } from '../post/Post';
+import { ProtoPost } from '../ProtoPost';
+
+export const PostList = ({ preparedPosts }) => (
+  <>
+    {preparedPosts.map(post => (
+      <div className="post" key={post.id}>
+        <Post post={post} />
```

Comment:
Same thing with spread operator here
```suggestion
        <Post post={...post} />
```